tasks/task267_paper_reviews_classification.json

```diff
@@ -0,0 +1,2425 @@
+{
+    "Contributors": "Mihir Parmar",
+    "Source": "Paper Reviews Dataset (dataset link- https://archive.ics.uci.edu/ml/datasets/Paper+Reviews)",
+    "Categories": "Classification",
+    "Definition": "In this task, you are given a paper review, and your job is to obtain a judgment of each review using one of the five orientation metrics: (1) very negative, (2) negative, (3) neutral, (4) positive, and (5) very positive. This attribute (orientation metric) represents the subjective perception of each review (i.e., how negative or positive the review is perceived when someone reads it).",
```

Comment:
Not sure what is the difference between this and the prior task.

Reply:
Difference (See Table 2 - https://sentic.net/wisdom2017fuentes.pdf): 

Although the scale is the same,

Evaluation metric suggests a reviewers' opinion towards paper, rather orientation suggests readers' (who are reading reviews) opinion (not reviewers) towards the paper. Both are different.

I added this difference very clearly in the definition of both tasks.